theme.json

```diff
@@ -246,7 +241,6 @@
 			},
 			"core/query-title": {
 				"typography": {
-					"fontFamily": "var(--wp--preset--font-family--system-font)",
 					"fontSize": "clamp(3.25rem, calc(3.25rem + ((1vw - 0.48rem) * 8.4135)), 3.625rem)",
 					"fontWeight": "300",
 					"lineHeight": "1.2"
```

Comment:
And this

Reply:
Thanks! Those settings weren't making any difference. I've removed them.